spec/v3/services/log/find_spec.rb

```diff
@@ -154,13 +175,61 @@
 
     describe 'it returns the correct content type' do
       example do
-        s3log.update_attributes(archived_at: Time.now)
+        s3log.update_attributes(archived_at: Time.now, archive_verified: true)
```

Comment:
Will this attribute be set on existing logs? I assume this is a field we didn't take into account before?